File: codewars/python/8kyu/multi_table.py
def multi_table(number: int):
    res = ''
    for i in range(1, 11):
        res += f'{i} * {number} = {i * number}\n'
    res = res.strip()
    return res


if __name__ == '__main__':
    print(multi_table(5))
    print(multi_table(1))

File: codewars/go/8kyu/multi_table.go
package main

import (
	"fmt"
)

func main() {
	fmt.Println(MultiTable(5))
	fmt.Println(MultiTable(1))
}

func MultiTable(number int) string {
	var res string
	for i := 1; i <= 10; i++ {
		res += fmt.Sprintf("%v * %v = %v", i, number, i*number)
		if i != 10 {
			res += "\n"
		}
	}
	return res
}
